espnet/asr/chainer_backend/asr.py

```diff
@@ -78,37 +78,48 @@ class CustomUpdater(training.StandardUpdater):
     def __init__(self, train_iter, optimizer, converter, device, accum_grad=1):
         super(CustomUpdater, self).__init__(
             train_iter, optimizer, converter=converter, device=device)
-        self.count = 0
+        self.forward_count = 0
```

Comment:
I feel changing from `count` to `forward_count` seems to have side effects. 
Do you want to fix the code of `accum_grad`?

Reply:
I did not find any side effects on the accum_grad with forward_count but I will check it once more. Could you explain me which possible effect  appear.?

Reply:
I'm sorry. This is my concern. 
I think `CustomUpdater` and`CustomParallelUpdater` are common components.
Is this modification necessary for the Transformer PR?
Actually, I'm not sure why this modification is necessary. (This is just comment.)

If this modification for `CustomUpdater` and`CustomParallelUpdater` is not related to Transformer method, I feel you can separate PR into different PRs.
But I'm not the main contributor of ESPNET. so I'm not sure we should separate PR or not.